sofiadinizms/projeto-es
# Feature/menu

## What I did
1. Created the new Menu
3. Created associated Screens

## How to test

1. Go to the project folder
3. In the terminal type: `cd nft-the-game`
4. Next `yarn && expo start`

Notes: make sure you have expo updated and a mobile device or simulator
